lib/sentry/application.ex

```diff
@@ -19,6 +19,12 @@ defmodule Sentry.Application do
         []
       end
 
+    # if Config.dsn() && Config.send_client_reports?() do
+    client_report_enabled =
+      if Config.send_client_reports?() do
+        [Sentry.ClientReport]
```

Comment:
Cheap enough to just start this anyway, and also allows for runtime changing of `:send_client_reports`. Let's always start `Sentry.ClientReport`.